<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrumBurpWindow</class>
 <widget class="QMainWindow" name="DrumBurpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DrumBurp</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="optionsFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="noteHeadGroupBox">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>Note Head</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="RadioButtonTeller" name="defaultNoteHeadButton">
            <property name="text">
             <string>Default</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="xNoteHeadButton">
            <property name="text">
             <string>x</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="bigXNoteHeadButton">
            <property name="text">
             <string>X</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="oNoteHeadButton">
            <property name="text">
             <string>o</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>o</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="bigONoteHeadButton">
            <property name="text">
             <string>O</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>O</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="plusNoteHeadButton">
            <property name="text">
             <string>+</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButtonTeller" name="gNoteHeadButton">
            <property name="text">
             <string>g</string>
            </property>
            <property name="buttonValue" stdset="0">
             <string>g</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="displayOptionsGroupBox">
         <property name="title">
          <string>Display Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="spacingLabel">
            <property name="text">
             <string>Horizontal Spacing</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>spaceSlider</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="spaceSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Vertical Spacing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>System Spacing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBelow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="widthLabel">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="widthGroupBox">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="fixedWidthCheckBox">
               <property name="layoutDirection">
                <enum>Qt::RightToLeft</enum>
               </property>
               <property name="text">
                <string>Fixed</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="widthSpinBox">
               <property name="minimum">
                <number>10</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>80</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="songPropertiesGroupBox">
      <property name="title">
       <string>Song Properties</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="songNameLabel">
         <property name="text">
          <string>Name</string>
         </property>
         <property name="buddy">
          <cstring>songNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="songNameEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="bpmLabel">
         <property name="text">
          <string>&amp;BPM</string>
         </property>
         <property name="buddy">
          <cstring>bpmSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="artistNameLabel">
         <property name="text">
          <string>&amp;Artist</string>
         </property>
         <property name="buddy">
          <cstring>artistNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="artistNameEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="tabberLabel">
         <property name="text">
          <string>&amp;Tabbed by</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
         <property name="buddy">
          <cstring>tabberEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="tabberEdit"/>
       </item>
       <item row="0" column="3">
        <widget class="QFrame" name="bpmFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSpinBox" name="bpmSpinBox">
            <property name="suffix">
             <string> bpm</string>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>154</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="ScoreView" name="scoreView">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::RubberBandDrag</enum>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExport_ASCII"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit DrumBurp</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExport_ASCII">
   <property name="text">
    <string>Export ASCII</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RadioButtonTeller</class>
   <extends>QRadioButton</extends>
   <header>Widgets.RadioButtonTeller_plugin</header>
  </customwidget>
  <customwidget>
   <class>ScoreView</class>
   <extends>QGraphicsView</extends>
   <header>Widgets.ScoreView_plugin</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>defaultNoteHeadButton</tabstop>
  <tabstop>xNoteHeadButton</tabstop>
  <tabstop>bigXNoteHeadButton</tabstop>
  <tabstop>oNoteHeadButton</tabstop>
  <tabstop>bigONoteHeadButton</tabstop>
  <tabstop>plusNoteHeadButton</tabstop>
  <tabstop>songNameEdit</tabstop>
  <tabstop>artistNameEdit</tabstop>
  <tabstop>bpmSpinBox</tabstop>
  <tabstop>tabberEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fixedWidthCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>widthSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>defaultNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bigXNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>oNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bigONoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plusNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gNoteHeadButton</sender>
   <signal>emitValue(QString)</signal>
   <receiver>scoreView</receiver>
   <slot>setNoteHead(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spaceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scoreView</receiver>
   <slot>horizontalSpacingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scoreView</receiver>
   <slot>verticalSpacingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>scoreView</receiver>
   <slot>systemSpacingChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
